.github/workflows/ci.yml

```diff
@@ -423,9 +423,10 @@ jobs:
   build-binary-linux:
     timeout-minutes: 10
     needs: determine_changes
-    if: ${{ github.repository == 'astral-sh/uv' && (needs.determine_changes.outputs.code == 'true' || github.ref == 'refs/heads/main') }}
-    runs-on:
-      labels: ubuntu-latest-large
+    #if: ${{ github.repository == 'astral-sh/uv' && (needs.determine_changes.outputs.code == 'true' || github.ref == 'refs/heads/main') }}
+    runs-on: ubuntu-latest
```

Comment:
This was just because i was testing in my personal fork to avoid messing with permissions on astral-sh/uv while i figure it out so i blanket removed everything custom